docs/troubleshooting.rst

```diff
@@ -118,6 +118,33 @@ crash with an ``ImportError`` similar to::
 
 To fix the error, change the name of the script to something other than ``bleak.py``.
 
+----------
+Linux Bugs
+----------
+
+Occasional "Not connected" errors on Raspberry Pi
+=================================================
+
+If you are using a Raspberry Pi and are seeing occasional "Not connected" errors,
```

Comment:
I assume this is using the built-in WiFi and Bluetooth on the newer RPis? Probably would be helpful to clarify that in the text.

Reply:
If what I'm suspecting is correct you're right, it shouldn't affect USB connected radios as they don't use the same chip. I'll see if it can be rephrased.